repository/inventory_repo.go

```diff
@@ -90,6 +90,7 @@ func (i InventoryRepo) BuildAssetListFromConfig(config []domain.ProxyConfig) (ma
 
 	for _, cfg := range config {
 		for _, env := range cfg.Environments {
+			//inventory := make(map[string]string)
```

Comment:
Guessing this was accidentally committed?

Reply:
thanks